src/js/src/components/SearchBox.vue

```diff
@@ -148,6 +151,14 @@ export default {
       updateShowModal:'updateShowModal',
       updateCurrentModal:'updateCurrentModal'
     }),
+    textAreaClick() {
+      console.log('clicks!')
+    },
+    getSizeOfTextArea() {
+      let textarea = document.getElementById('query')
+      textarea.style.height = '1px'
+      textarea.style.height = (textarea.scrollHeight) + 'px'
+    },
     selectSearchMethod(selected) {
       console.log(selected)
```

Comment:
Still need this?

Reply:
Yep. This is what makes sure the searchbox grows when a new line is added.

Reply:
removed the console.log tho. If that was what you were refering to :D

Reply:
Yes it was :)